### ISSUE TITLE
Section 2.4 (comment 9)

### ISSUE THREAD

**Issue by LPardue** [quicwg/ops-drafts#533]
[Section 2.4](https://www.rfc-editor.org/authors/rfc9312.html#section-2.4): We suggest rephrasing this sentence in order to clarify the exact minimum packet size. Does the suggested text change the meaning of the original sentence?  

Original:

       A network path needs to be able to forward at least this size of
       packet for QUIC to be used.  

Perhaps: 
   A network path needs to be able to forward at least the size of
   an Initial packet in order for QUIC to be used.

**Comment by mirjak** [comment]
Yes, I think that alters the meeting because it seems to indicate that networks only need to enables initial packets to be this size.

I guess this could be better:

A network path needs to be able to forward at least this packet size for QUIC to be used.  

Or even better:

A network path needs to be able to forward packets of at least this size for QUIC to be used.

**Comment by britram** [comment]
+1 to your "even better"
